src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1164,17 +1164,17 @@ void *os::dll_load(const char *filename, char *ebuf, int ebuflen) {
   void* result = nullptr;
   unsigned long buffer_length = strlen(filename);
   int extension_length = 3;
-  char* file_path=NEW_C_HEAP_ARRAY(char, buffer_length + extension_length + 1, mtInternal);
+  char* file_path = NEW_C_HEAP_ARRAY(char, buffer_length + extension_length + 1, mtInternal);
   strncpy(file_path,filename, buffer_length + 1);
```

Comment:
Why not using 
`char* file_path = os::strdup (filename);`
which would replace lines 1167+1168
and use the corresponding 
`os::free (file_path);`
at the end

Reply:
Ok. Any performance advantage to using that ?

Reply:
No, I do not believe that it has performance advantage, but I think it is simpler to understand.